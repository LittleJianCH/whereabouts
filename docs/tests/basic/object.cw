query (abc) {
  equation abc = {
    a: "a",
    b: "b",
    c: "c",
  }
}

query (a, b, c) {
  equation { a, b, c } = {
    a: "a",
    b: "b",
    c: "c",
  }
}

query (empty) {
  equation empty = {}
}
