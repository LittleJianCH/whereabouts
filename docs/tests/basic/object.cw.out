{ "solutions": [{ "abc": { "a": "a", "b": "b", "c": "c" } }] }
{ "solutions": [{ "a": "a", "b": "b", "c": "c" }] }
